lib/vtls/gtls.c

```diff
@@ -664,7 +664,11 @@ gtls_connect_step1(struct connectdata *conn,
   }
 
   /* Initialize TLS session as a client */
-  init_flags = GNUTLS_CLIENT | GNUTLS_FORCE_CLIENT_CERT;
+  init_flags = GNUTLS_CLIENT;
+
+#if defined(GNUTLS_FORCE_CLIENT_CERT)
```

Comment:
Maybe just do #ifdef as that's how we normally check for the presence of a single define.

Reply:
Whilst I'm the first to admit that I'm a fully fledged member of the `#if everywhere` club, I do "try" and use the style of the code that is around me.

As such I noted the following on line 673 and kept the styling the same:

````
#if defined(GNUTLS_NO_TICKETS)
````

Reply:
I think "fixing the code while changing it" is better than sticking to the "wrong" style...